scripts/profile-tcl.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# Variables
+build_id=""
+branch="develop"
+arch="gfx900"
+compiler="amdclang++"
+
+# Constants
+base_image="compute-artifactory.amd.com:5000/rocm-plus-docker/compute-rocm-dkms-no-npi-hipclang"
```

Comment:
Consider removing and require setting it in the users environment.

Reply:
Sounds good, I'll setup an env var called something like REGISTRY_IMAGE